api_test.sh

```diff
@@ -30,7 +30,7 @@ case "$1" in
     DB_CONTAINER="func-mysql-test"
     docker rm -fv ${DB_CONTAINER} || echo No prev mysql test db container
     docker run --name ${DB_CONTAINER} -p 3306:3306 -e MYSQL_DATABASE=funcs -e MYSQL_ROOT_PASSWORD=root -d mysql
-    sleep 15
+    sleep 5
```

Comment:
we should "loop curl/connect with sleep 1" to wait for that TCP port instead of big sleep here IMHO.

Reply:
there's an alternative, start datastore container, build binary (takes 1m), by that time datastore container will definitely be up and running.

Reply:
sounds good, not a big deal, I wouldn't spend too much time on this. :-)